sechub-commons-security-spring/src/test/java/com/mercedesbenz/sechub/spring/security/LoginOAuth2SuccessHandlerTest.java

```diff
@@ -65,7 +68,10 @@ void on_authentication_success_sends_a_valid_redirect_containing_the_encrypted_a
         Instant now = Instant.now();
         when(oAuth2AccessToken.getIssuedAt()).thenReturn(now);
         /* setting this should make sure that the default expiry (1 hour) is not used */
-        when(oAuth2AccessToken.getExpiresAt()).thenReturn(now.plusSeconds(expiry.toSeconds()));
+        Instant nowPlusOneSecond = now.plusSeconds(expiry.toSeconds());
```

Comment:
```java
Duration expiry = Duration.ofMinutes(1);
Instant nowPlusOneSecond = now.plusSeconds(expiry.toSeconds());
```
==> means:
nowPlusOneSecond = now + oneMinuteInSeconds 

==> naming ... is not a "nowPlusOneMinute" instead?